app/policies/team_policy.rb

```diff
@@ -19,6 +19,18 @@ def destroy?
     is_admin?
   end
 
+  def manage_users?
+    is_admin?
+  end
+
+  def new_enrollment?
+    is_admin?
```

Comment:
I think you could use `manage_users?` here and inside `create_enrolment?`, what do you think? Because they'll always follow the same rules and then it'll be easier to change it in the future if needed.